perses/data/cdk2-example/cdk2_setup_neq.yaml

```diff
@@ -0,0 +1,67 @@
+# Relative path to protein PDB file (with no missing heavy atoms)
+protein_pdb: CDK2_fixed_nohet.pdb
+
+# Relative path to ligand SDF file
+ligand_file: CDK2_ligands.sdf
+
+# Indices of the ligands in the ligand_file between which we should compute a relative free energy
+# (zero-indexed)
+old_ligand_index: 14
+new_ligand_index: 15
+
+# OpenMM ffxml force field files installed via the openmm-forcefields package
+# for biopolymers and solvents.
+# Note that small molecule force field files should NOT be included here.
+forcefield_files:
+    - amber/protein.ff14SB.xml # ff14SB protein force field
+    - amber/tip3p_standard.xml # TIP3P and recommended monovalent ion parameters
+    - amber/tip3p_HFE_multivalent.xml # for divalent ions
+
+# Small molecule force field
+# We recommnd one of ['gaff-1.81', 'gaff-2.11', 'smirnoff99Frosst-1.1.0', 'openff-1.0.0']
+small_molecule_forcefield: gaff-2.11
```

Comment:
what would happen if a small molecule FF file was provided in `forcefield_files` too?

Reply:
Either the `small_molecule_forcefield` will win out or an Exception will be thrown if atom types are redefined by the selected force field. I'd advise against including a small molecule FF in the `forcefield_files`.